cves/kernel/CVE-2020-11668.yml

```diff
@@ -278,8 +275,9 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    This vulnerability dealt with resource management
```

Comment:
Elaborate on why resource management is not sandboxing or perhaps give more thorough explanation on the reasoning